pkg/apis/workspaces/v1alpha2/component_image_dockerfile.go

```diff
@@ -58,7 +58,8 @@ type Dockerfile struct {
 }
 
 type DockerfileDevfileRegistrySource struct {
-	// Id in a devfile registry that contains a Dockerfile
+	// Id in a devfile registry that contains a Dockerfile. The src in the OCI registry
```

Comment:
Nit: the commit msg only references adding tests but this commit updates documentation and regenerates files. Doesn't need to be fixed but we should avoid this in the future.

Reply:
I suppose it's unclear from just the comment, but I'm referring to d918244a9a718e243695cd59a78f0e1e28fe1dc5

Reply:
Sorry, noted.